nym-api/nym-api-requests/src/models.rs

```diff
@@ -115,19 +116,45 @@ pub struct MixnodeStatusResponse {
     pub status: MixnodeStatus,
 }
 
-#[derive(Clone, Debug, Default, Serialize, Deserialize, JsonSchema)]
+#[derive(Clone, Debug, Default, Serialize, Deserialize, JsonSchema, ToSchema)]
 pub struct NodePerformance {
+    #[schema(value_type = String)]
     pub most_recent: Performance,
+    #[schema(value_type = String)]
     pub last_hour: Performance,
+    #[schema(value_type = String)]
     pub last_24h: Performance,
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, JsonSchema)]
+#[derive(ToSchema)]
+#[schema(title = "MixNodeDetails")]
+pub struct MixNodeDetailsSchema {
+    // TODO dz copy from cosmwasm crate
+    /// Basic bond information of this mixnode, such as owner address, original pledge, etc.
+    pub bond_information: String,
+
+    // TODO dz copy from cosmwasm crate
+    /// Details used for computation of rewarding related data.
+    pub rewarding_details: String,
+
+    // TODO dz copy from cosmwasm crate
+    /// Adjustments to the mixnode that are ought to happen during future epoch transitions.
+    pub pending_changes: String,
+}
+
+#[derive(ToSchema)]
+#[schema(as = String, title = Decimal)]
+pub struct DecimalSchema;
```

Comment:
do you need a separate `DecimalSchema`? iirc. for all intents and purposes it behaves like string when serialised

Reply:
didn't end up using this, removed